web/modules/custom/joinup_core/joinup_core.module

```diff
@@ -958,23 +959,32 @@ function joinup_core_user_predelete(EntityInterface $entity) {
 }
 
 /**
- * Invalidate Joinup user membership cache tags after a user is updated.
+ * Implements hook_ENTITY_TYPE_update().
  *
- * @param \Drupal\Core\Entity\EntityInterface $user
- *   The user entity.
+ * Invalidates group membership list cache tags after a user is (un)blocked.
  *
- * @see joinup_core_og_membership_invalidate_group_tags()
+ * If a user gets blocked in the entire site we no longer want to include this
+ * user in the collection membership counts. We handle this use case here in
+ * Joinup Core since Organic Groups currently only handles users that are
+ * blocked on group level through the membership state; at this time OG is
+ * not acting on users that are blocked on user level.
```

Comment:
This raises an interesting OG business model question. I.e. _What is the fate of user that is 'OG active' but blocked?'. I think this philosophical issue deserves a ticket upstream.